### PR TITLE
Server and components

### PR DESCRIPTION
**Pull request by rhahn07**
hi guys. here's where i'm at now

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+  this.state = {
+    listOfReviews: [],
+    reviewers: []
+  };  
+  }
```
Comment (KiltedDrummer) [inline client/src/components/App.jsx]: nit picky: can you fix this indent? :P 
linters have been drilled into my brain.

File: client/src/components/ReviewList.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+// const Container = styled.container`
+//   display: grid;
+//   grid-template-columns: 1fr 2fr;
+// `  
+// const ReviewItem = styled.item`
+//   display: grid;
+//   grid-template-columns: 1fr 2fr;
+// `
```
Comment (KiltedDrummer) [inline client/src/components/ReviewList.jsx]: this still in progress or old code to remove?

File: test/test.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  page = await browser.newPage();
+  await page.setViewport({ width, height });
+});
+
+afterAll(() => {
+  browser.close();
+});
+
+describee('search function' () => {
+
```
Comment (KiltedDrummer) [inline test/test.js]: I'm guessing tests are to come later? 
also please  fix spelling on 'describee'

**Review by KiltedDrummer — CHANGES_REQUESTED** [review]
Overall looks like it's coming together fine.
Watch out for your presets in your .babelrc though.
I think you need to be using "@bebel/preset-env" same for react.

I don't think you have any errors from it right now because your babelrc file is named wrong. I struggled for way too long on this, but it should be '.babelrc' not babel.rc', might give you some new errors, but better now than later!

File: client/src/components/App.jsx (hunk 1 above)
Comment (rhahn07) [reply client/src/components/App.jsx]: will do! thanks for the catch. i've installed linters, but not sure if they're working?

File: client/src/components/ReviewList.jsx (hunk 2 above)
Comment (rhahn07) [reply client/src/components/ReviewList.jsx]: i'll delete that. early experimentation

File: test/test.js (hunk 3 above)
Comment (rhahn07) [reply test/test.js]: got it. thanks!

**Comment by rhahn07** [comment]
Thanks for looking it over, Nick. I've made the changes you suggested

**Comment by KiltedDrummer** [comment]
mind fixing your babelrc file? 
other than that all looks god

**Comment by rhahn07** [comment]
thanks nick i went ahead and made that change

**Review by KiltedDrummer — APPROVED** [review]

**Review by brandonjunus — APPROVED** [review]

**Comment by brandonjunus** [comment]
It honestly looks like you've written a really clean code here Rich! I was trying to find things that I would fix/change, but it all looks pretty clean. You're good to merge and keep working!

**Pull request merged**